Source/WebCore/style/StyleExtractorCustom.h

```diff
@@ -1417,12 +1417,18 @@ inline void ExtractorCustom::extractLetterSpacingSerialization(ExtractorState& s
 
 inline Ref<CSSValue> ExtractorCustom::extractWordSpacing(ExtractorState& state)
 {
-    return ExtractorConverter::convertLength(state, state.style.computedWordSpacing());
+    float zoomedSpacing = state.style.fontCascade().wordSpacing();
```

Comment:
this will compute with zoom on every call.

we need to somehow use computedWordSpacing but that returns unzoomed values